### ISSUE TITLE
Fastmail sieve tester linked doesn't work without stop; in if clauses

### ISSUE THREAD

**Issue by JMoVS** [Uberspace/manual#370]
Hi,

in my testing for sieve email filters I found out that the helpfully linked fastmail sieve tester does not work with `fileinto :copy` inside if clauses if those if clauses are not ended with a `stop`

A simple note could be added that states something like:

*Beware when using the fastmail sieve tester that your if condition checks should still work on uberspace without the supplied `stop;` keyword, even though the fastmail checker will not show a result.*

**Comment by nichtmax** [comment]
But even though it works with our servers without the `stop;` there _should_ be the `stop;`, right? I'm having a bit of a hard time to document something that works out of the specification 😅

**Comment by JMoVS** [comment]
no, the `stop` actually prohibits some use cases as it stops all processing, even if there were other else if clauses. So if I want to check for multiple things or do mutliple things with an email, it *must* not contain a `stop` ;-)
